VP-60: Add Velocloud documentation to main docs menu

Bonrob2 opened the pull request:
Needed to update this content so that the Velocloud docs will appear on the docs site. Creating draft PR as will need @christianpape to update the Velocloud repo to have a release branch.

Bonrob2 committed:
VP-60: Add Velocloud documentation to main docs menu

VP-60: Add Velocloud documentation to main docs menu - specified location for velocloud docs to add them to the main TOC and docs website.

Bonrob2 committed:
VP-60: Add Velocloud documentation to main docs menu

VP-60: Add Velocloud documentation to main docs menu - added entry for Velocloud to ensure the content appears in the main docs.

christianpape commented:
> Needed to update this content so that the Velocloud docs will appear on the docs site. Creating draft PR as will need @christianpape to update the Velocloud repo to have a release branch.

I added a release-1.x branch.

Bonrob2 commented:
> I added a release-1.x branch.

Thank you!

indigo423 commented on antora-playbook.yml:
```diff
@@ -79,7 +79,12 @@ content:
-  - url: https://github.com/OpenNMS/opennms-provisioning-integration-server.git
+    - url: https://github.com/OpenNMS/opennms-velocloud-plugin
+    start_path: docs
+    branches: main
```
He made a branch named "release-1.x" you need to reference it here instead of "main". It seems to me he doesn't want or just simply can't make a release. What will happen the version number will be shown as [1.0.0-SNAPSHOT](https://github.com/OpenNMS/opennms-velocloud-plugin/blob/99792066c0ed0adb49ac37c9bd105cae415e3d42/docs/antora.yml#L2). If this is ok you can for it.

Bonrob2 replied:
For sustainability, I don't think that will work, as we need to make sure the Velocloud docs update with each major release as well. 
 
If I reference `release-*` will that work? Or is there something @christianpape needs to do so we can set up the Antora syntax correctly?

Bonrob2 committed:
VP-60: Add Velocloud documentation to main docs menu

Updated to remove tag and use the main branch. Will add version tag when Velocloud is released.

Bonrob2 committed:
VP-60: Add Velocloud documentation to main docs menu

Updated error in syntax.

Bonrob2 committed:
VP-60: Add Velocloud documentation to main docs menu

Updated bad indentation syntax error.

Bonrob2 committed:
VP-60: Add Velocloud documentation to main docs menu

Fixed second indentation syntax error.

Bonrob2 commented:
@indigo423 , could you approve this PR?

Bonrob2 commented:
@christianpape , would you be able to approve this PR?

indigo423 reviewed (approved):
Built the docs locally and they get generated for 1.0.0-SNAPSHOT. 👍 I think you can safely remove the "DRAFT" mode for this PR and make it a real one so you can merge it.
